.github/workflows/ci.yaml

```diff
@@ -11,3 +11,19 @@ on:
 jobs:
   nox:
     uses: ./.github/workflows/reusable-nox.yml
+    
+  check:
+    if: always()
+
+    needs:
+      - nox
+
+    runs-on: Ubuntu-latest
+
+    steps:
+    - name: Decide whether the needed jobs succeeded or failed
+      uses: re-actors/alls-green@release/v1
+      with:
+        allowed-failures: docs, linters
+        allowed-skips: non-voting-flaky-job
+        jobs: ${{ toJSON(needs) }}
```

Comment:
@x1101 Just taking a cursory look through for now. I'll take another look tomorrow. Could you please add an extra line at the end of this file and the other one?